spec/cypress/integration/attendanceView.spec.js

```diff
@@ -0,0 +1,107 @@
+import faker from 'faker'
+import { createSelector } from '../utils'
+import dayjs from 'dayjs'
+import weekday from 'dayjs/plugin/weekday'
+
+dayjs.extend(weekday)
+
+const { name, internet } = faker
+const firstName = name.firstName()
+const fullName = name.findName(firstName)
+const email = internet.email(firstName)
+const password = internet.password()
+const checkIn = new Date(Date.now() - 21600000)
```

Comment:
Can we just comment to document how much time this is subtracting in human-readable terms?  i.e.
```
// Date now - 6 hours
```
or whatever it is

Reply:
yes!